bag/datasets/brk/batch.py

```diff
@@ -834,12 +836,12 @@ def tasks(self):
             ImportKadastraleSectieTaskLines(self.brk_shp, self.stash),
```

Comment:
Moet het niet `self.brk_shp_with_subj` zijn?

Reply:
Inderdaad, verkeerde gecopy-paste...

Reply:
Variabele ook even weggehaald, aangezien deze niet nodig is